src/AutoRest.CSharp/Mgmt/Decorator/Node.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AutoRest.CSharp.Output.Models.Types
+{
+    internal class Node
```

Comment:
This type can be nested into InheritanceChoser or replaced with` KeyVaultPair<>`